code/game/machinery/spaceheater.dm

```diff
@@ -265,6 +257,14 @@
 	panel_open = TRUE
 	update_appearance()
 
+/obj/machinery/space_heater/proc/toggle_space_heater_state()
```

Comment:
You could prob just have this be `toggle_on` or something, but it's not a huge deal.

Reply:
I actually considered that, but the same proc can also toggle it off, and `toggle_state` is a global.

Reply:
`toggle_heat`?

Reply:
I considered that too, but since that could be confused for changing the mode of the space heater from one of auto/heat/cool I didn't want to create confusion with its current functionality.  Any other ideas?  I would be happy to change it to something more concise that didn't create confusion.

Reply:
I changed it to `toggle_power` .  I think that will work without creating confusion.  Thoughts?

Reply:
Seems good to me